<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetAssistantWidget</class>
 <widget class="QWidget" name="NetAssistantWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>网络设置</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="Label">
           <property name="text">
            <string>设置模式</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="cb_mode">
           <item>
            <property name="text">
             <string>TCP客户端</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>TCP服务端</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>UDP</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_ip">
           <property name="text">
            <string>服务器IP:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="edit_ip">
           <property name="inputMask">
            <string>000.000.000.000</string>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_port">
           <property name="text">
            <string>服务器端口：</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="edit_port">
           <property name="placeholderText">
            <string>0-65536</string>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="btn_connect">
         <property name="text">
          <string>连接服务器</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTextEdit" name="edit_recv"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layout_client" stretch="0,1,0,0">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>本地ip地址：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="local_ip"/>
         </item>
         <item>
          <widget class="QLabel" name="label_local_port">
           <property name="text">
            <string>本地端口</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="local_port">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layout_server" stretch=""/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QTextEdit" name="edit_send"/>
         </item>
         <item>
          <widget class="QPushButton" name="btn_send">
           <property name="text">
            <string>发送</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
